tests/openwisp2/settings.py

```diff
@@ -85,6 +85,7 @@
 STATIC_URL = '/static/'
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 OPENWISP_USERS_AUTH_API = True
+OPENWISP_IPAM_API = True
```

Comment:
let's avoid this setting, some features of the API is used by the admin as well so it doesn't make sense to turn it off

Reply:
Okay